buildkite/bazelci.py

```diff
@@ -2254,7 +2254,7 @@ def set_env_var(config_key, env_var_name):
     if not (
         is_pull_request()
         or use_but
-        or os.getenv("BUILDKITE_BRANCH") != "master"
+        or (os.getenv("BUILDKITE_BRANCH") != "master" and os.getenv("BUILDKITE_BRANCH") != "stable")
```

Comment:
While we're here, should be also add the check for `main`?

Reply:
I'd rather wait for the first project to have it. This thing can produce problems if multiple branches are active.

Reply:
I see. Yeah, we have to be careful. Looks like rules_pkg has already switched to `main` - maybe that's a good enough reason?

Reply:
It is indeed. Done.